### PR TITLE
Certificate: Fix warnings in unit tests

### PR DESCRIPTION
**Pull request by mjansenDatabay**
Mantis Issue: https://mantis.ilias.de/view.php?id=29248

### CODE REVIEW

File: Services/Certificate/test/ilCoursePlaceholderDescriptionTest.php [hunk 1]
```diff
@@ -75,7 +75,6 @@ public function testPlaceholderDescriptions()
 
         $customUserPlaceholderObject = $this->getMockBuilder('ilObjectCustomUserFieldsPlaceholderDescription')
             ->disableOriginalConstructor()
-            ->setMethodsExcept(['getPlaceholderDescriptions'])
```
Comment (mjansenDatabay) [inline Services/Certificate/test/ilCoursePlaceholderDescriptionTest.php:78]: @M-Barz I'll had to remove this, maybe you want to review this :-).

File: Services/Certificate/classes/Placeholder/Description/class.ilCoursePlaceholderDescription.php [hunk 2]
```diff
@@ -57,7 +57,7 @@ public function __construct(
         }
         $this->customUserFieldsPlaceholderDescriptionObject = $customUserFieldsPlaceholderDescriptionObject;
 
-        $customUserFieldsPlaceholderHtmlDescription = $this->customUserFieldsPlaceHolderDescriptionObject->getPlaceholderDescriptions();
+        $customUserFieldsPlaceholderHtmlDescription = $this->customUserFieldsPlaceholderDescriptionObject->getPlaceholderDescriptions();
```
Comment (mjansenDatabay) [inline Services/Certificate/classes/Placeholder/Description/class.ilCoursePlaceholderDescription.php:60]: @M-Barz This is a strange one. I cannot spot the difference (maybe a non-printable whitespace char), but this 'change' was essential to get the test succeeding.

**Comment by mjansenDatabay** [comment]
@M-Barz I changed the classes you modfied for the ILIAS 7 feature. Maybe you'll want to have a look:

- https://github.com/ILIAS-eLearning/ILIAS/pull/3021/files#diff-118e0d8e6e7b42cd8822b984297b65135340ea4bcb1e12691b66db3be6df28ed (very strange changes)
- https://github.com/ILIAS-eLearning/ILIAS/pull/3021/files#diff-0fdcd9d28d93c8b3e2270001deaa436ffc40ea0c6b47260880799d1caffc2994

**Review by M-Barz — APPROVED** [review]
@mjansenDatabay Sorry about that! I didn't realise I was the actual reviewer, first time and all.
That being said, this looks good to me. Thanks a lot! :)

**Pull request merged**